simpeg_drivers/constants.py

```diff
@@ -131,187 +67,143 @@
         "tooltip": "Apply a non-homogeneous offset to survey object from radar channel.",
         "optional": True,
         "parent": "data_object",
-        "value": None,
+        "value": "",
         "enabled": False,
         "verbose": 3,
+        "visible": False,
+    },
+    "gps_receivers_offset": "",
+    "potential_channel": {
```

Comment:
What's this doing in here?  Should only be specified in the electrical module constants files, right?

Reply:
Yep

Reply:
Yes